dbt_sugar/core/ui/cli_ui.py

```diff
@@ -169,13 +173,57 @@ def _validate_question_payload(self) -> None:
         self._is_valid_question_payload = True
 
     @staticmethod
-    def _iterate_through_columns(cols: List[str]) -> Dict[str, str]:
+    def _iterate_through_columns(
+        cols: List[str], ask_for_tests: bool = True
+    ) -> Mapping[str, Mapping[str, Union[str, List[str]]]]:
+        """Iterates through a provided list of columns collects documentation info.
+
+        More specifically it asks for the following
+        - a column documentation
+        - whether user wants to add tests
+            - and if so which tests to add from a list of pre-defined/aggreed tests.
+
+        Args:
+            cols (List[str]): List of columns to collect description and tests info from.
+            ask_for_tests (bool, optional): When set to true UI will ask for whether the user wants
+            to add tests for each of the columns. Defaults to False.
```

Comment:
Bastien, here you say default to false but in the method the default is true.

Reply:
Yeah i wrote the docstring and then I changed my mind. What do you think would make more sense?

Reply:
the test in True, I think.

Reply:
Ok I'll leave it as is and update the docstring!